src/dataflow/sink/kafka.rs

```diff
@@ -22,6 +23,21 @@ use expr::GlobalId;
 use interchange::avro::{DiffPair, Encoder};
 use repr::{RelationDesc, Row};
 
+#[derive(Clone)]
+pub struct SinkProducerContext;
+
+impl ClientContext for SinkProducerContext {}
+impl ProducerContext for SinkProducerContext {
+    type DeliveryOpaque = ();
+
+    fn delivery(&self, result: &DeliveryResult, _: Self::DeliveryOpaque) {
+        match result {
+            Ok(_) => (),
+            Err((e, _)) => error!("received error while writing to kafka sink topic: {}", e),
```

Comment:
Perhaps makes sense to print out `msg.topic()`, too, since there might be multiple sinks in play?

Reply:
didn't know we had that - done!